pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.ohdsi</groupId>
     <artifactId>standardized-analysis-specs</artifactId>
-    <version>1.4.0</version>
+    <version>1.5.0</version>
```

Comment:
This should be -SNAPSHOT for our dev branch. We remove the -SNAPSHOT for release.